serverless/event_workflows/serverless-channels.adoc

```diff
@@ -12,18 +12,28 @@ image::serverless-event-channel-workflow.png[Channel workflow overview]
 
 After events have been sent to a channel from an event source or producer, these events can be sent to multiple Knative services, or other sinks, by using a subscription.
 
-include::modules/serverless-supported-channel-types.adoc[leveloffset=+1]
-// include::modules/serverless-inmemorychannel.adoc[leveloffset=+2]
-// comment out for now - this information isn't necessary while only default channel type is available, and should be moved to an admin guide anyway.
+{ServerlessProductName} supports `InMemoryChannel` and `KafkaChannel` channels for development use.
+
+:FeatureName: Apache Kafka on {ServerlessProductName}
+include::modules/technology-preview.adoc[leveloffset=+2]
+
+The following are limitations of `InMemoryChannel` type channels:
+
+* No event persistence is available. If a pod goes down, events on that pod are lost.
+* InMemoryChannel type channels do not implement event ordering, so two events that are received in the channel at the same time can be delivered to a subscriber in any order.
+* If a subscriber rejects an event, there are no re-delivery attempts. Instead, the rejected event is sent to a `deadLetterSink` if this sink exists, or is dropped.
```

Comment:
InMemoryChannel does support re-delivery if they are configured in "common delivery parameters" in a Subscription or a Broker (  https://knative.dev/docs/eventing/event-delivery/#common-delivery-parameters )